content/projects/gsoc/2025/project-ideas/complete-alternative-jenkins-io-build-retooling.adoc

```diff
@@ -0,0 +1,123 @@
+---
+layout: gsocprojectidea
+title: "Complete build retooling of jenkins.io"
+goal: "Using alternative tooling with Gatsby and Antora to build the Jenkins static site and provide versioned Jenkins documentation"
+category: Tools
+year: 2025
+status: published
+sig: documentation
+skills:
+- Web development
+- AsciiDoc
+- Static website tooling
+- Documentation
+- Website retooling
+mentors:
+- "krisstern"
+- "gounthar"
+- "vandit1604"
+links:
+  meetings: /projects/gsoc/#office-hours
+---
+
+=== Background
+
+The link:/[jenkins.io] website is generated as a static website using Awestruct from AsciiDoc sources, YAML data files, and HAML templates stored in GitHub.
+One of the drawbacks of the current build method is that the technical documentation is not product version bound.
+It is thus not possible to view the documentation for a given Jenkins version.
+Only the latest can be viewed.
+This can lead to unnecessary confusion and is a worse experience than many other documentation sites like the git site, FreeBSD, and others..
```

Comment:
```suggestion
The link:/[jenkins.io] website is generated as a static website using Awestruct from AsciiDoc sources, YAML data files, and HAML templates stored in GitHub.
One of the drawbacks of the current build method is that the technical documentation is not product version bound.
It is thus not possible to view the documentation for a given Jenkins version.
Only the latest can be viewed.
This can lead to unnecessary confusion and is a worse experience than many other documentation sites like the git site, FreeBSD, and others...
```